javalanglib/src/main/scala/java/lang/Long.scala

```diff
@@ -114,6 +114,48 @@ object Long {
     }
   }
 
+  // Ported from https://github.com/gwtproject/gwt/blob/master/user/super/com/google/gwt/emul/java/lang/Long.java
+  def toString(value: scala.Long, intRadix: Int): String = {
+    var _value: scala.Long = value
+
+    if (intRadix == 10 || intRadix < Character.MIN_RADIX || intRadix > Character.MAX_RADIX) {
+      toString(value)
+    }
+    else if (Integer.MIN_VALUE <= _value && (_value <= Integer.MAX_VALUE)) {
+      Integer.toString(_value.toInt, intRadix)
+    }
+    else {
+      def forDigit(digit: Int): Char = {
+          val overBaseTen = digit - 10
+          (if (overBaseTen < 0) '0' + digit else 'a' + overBaseTen).toChar
+      }
```

Comment:
Unfortunately, not possible. The java implementation specifically split forDigit as the one here has no range checks,  ```Character.forDigit``` does. If its OK, I keep as is - but add the missing comment as to why its needed